### PR TITLE
rm meaningless ready task

### PR DESCRIPTION
**Pull request by hzxuzhonghu**

### CODE REVIEW

File: src/app.rs [hunk 1]
```diff
@@ -42,7 +42,6 @@ pub async fn build_with_cert(
     let (drain_tx, drain_rx) = drain::channel();
 
     let ready = readiness::Ready::new();
-    let proxy_task = ready.register_task("proxy listeners");
```
Comment (hzxuzhonghu) [inline src/app.rs:45]: L50 `workload manager` has registered

Comment (howardjohn) [reply src/app.rs:45]: This is for the proxy listeners are bound which hapens at L75

**Review by howardjohn — COMMENTED** [review]
Its not clear to me why we do not need these

File: src/app.rs (hunk 1 above)
Comment (hzxuzhonghu) [reply src/app.rs:45]: not sure i understand

Comment (hzxuzhonghu) [reply src/app.rs:45]: I think there is only one task waiting for workloads discovery

Comment (howardjohn) [reply src/app.rs:45]: This task isn't waiting for workload discovery though, its waiting until we bind the listeners to ports (15008, 15001, etc)

Comment (hzxuzhonghu) [reply src/app.rs:45]: Not make much sense to me, if they bind failed, the process will exit.

Comment (howardjohn) [reply src/app.rs:45]: but we don't want to mark as ready before then

Comment (hzxuzhonghu) [reply src/app.rs:45]: Understand what you mean, will keep this

**Review by howardjohn — APPROVED** [review]

**Pull request merged**